simcoe-adobe/src/main/java/com/prolificinteractive/adobe/AdobeProvider.kt

```diff
@@ -0,0 +1,70 @@
+package com.prolificinteractive.adobe
+
+import android.content.Context
+import com.adobe.mobile.Analytics
+import com.adobe.mobile.AudienceManager
+import com.prolificinteractive.simcoe.Properties
+import com.prolificinteractive.simcoe.Success
+import com.prolificinteractive.simcoe.TrackingResult
+import com.prolificinteractive.simcoe.trackers.EventTracking
+import com.prolificinteractive.simcoe.trackers.PageViewTracking
+import com.prolificinteractive.simcoe.trackers.TimedEventTracking
+
+/**
+ * Creates a new adobe analytics provider instance.
+ */
+
+class AdobeProvider(private val token: String ="", private val appName: String) :
```

Comment:
I am gonna be removing the token. I am having trouble getting the project to build, it keeps complaining that the prolific-adobe package does not exist. was hoping to get some help on it